runtime/lua/vim/treesitter/health.lua

```diff
@@ -17,7 +17,7 @@ function M.check()
 
   for _, parser in pairs(parsers) do
     local parsername = vim.fn.fnamemodify(parser, ':t:r')
-    local is_loadable, ret = pcall(ts.language.require_language, parsername)
```

Comment:
`require_language` returned `true` if no error occurred, unlike `add`. If this is intentional, the check in line 22 has to be adapted.